Week1/homework/js-exercises/logString.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+ let myString = "Manolis Protopsaltis"
+ console.log("my name");
```

Comment:
Correct in plain and speaking human language. But in computer language, a `variable` is a placeholder with a `name` that actually holds a `value`.

Here the `variable` is the `myString` and its `value` is `Manolis Protopsaltis` which is a type of string, and true, it's your name! ;-D

Therefore, the answer here is: `console.log('The value of the string is "Manolis Protopsaltis"');`